### ISSUE TITLE
Translation: date and date time formats are not localized

### ISSUE THREAD

**Issue by xthiago** [podverse/podverse-web#935]
**Describe the bug**
Date and DateTime formats are not localized (at least for pt-BR).

**To Reproduce**
1. Go to a page where a date or date time is displayed in pt-BR.
4. See the format.

**Expected behavior**
Dates are expected to be displayed in format `d/m/Y` and date time in `d/m/Y H:i`. 

*`m` Month can be displayed as number or as localized string.

Examples of valid formats:

- date
  - 25/set/2020
  - 25/setembro/2020
  - 25 de setembro de 2020
- time
  - 20:01 (without seconds)
  - 20:01:22
  - 20h01m22s - alternative

We may have to check which format fits better on each app place :)

** I think we don't have to display seconds for

**Screenshots**

![image](https://user-images.githubusercontent.com/720151/189644095-e0e2ed85-58e1-4f4e-a074-102bf0fb7b7a.png)

![image](https://user-images.githubusercontent.com/720151/189644119-04445345-d305-43aa-9672-891d7c548382.png)

**Comment by xthiago** [comment]
I just remembered that my operating system is in English, and I'm using USA formats. 

Until tomorrow, I will check how the website is displayed on my wife's  laptop. She uses Brazilian format.

**Comment by xthiago** [comment]
Bug confirmed. 

I accessed podverse website from a Windows in Brazilian Portuguese – including number/date formats.

**Comment by mitchdowney** [comment]
@xthiago thanks for opening this. Would you be able to work on this? I believe we just need to add conditional format change when the lang is pt-br in this function: https://github.com/podverse/podverse-web/blob/develop/src/lib/utility/date.ts#L18

I can make the code change myself if you could confirm what the `moment` format needs to be.

**Comment by xthiago** [comment]
> Would you be able to work on this?

Yes. I can work with this issue on the weekend. Then I'm able to test different formats to pick the one who fits perfectly in each page.

**Comment by mitchdowney** [comment]
We are archiving all Github issues that pertain to the previous generation Podverse. The Podverse dev team is currently 100% focused on the next generation app, and will be only working on next gen Github issues going forward.

If you would like to see a work-in-progress demo of the upcoming Podverse website, please visit https://alpha.podverse.fm. Our tentative release date is Spring 2026.

If you're a developer who may be interested in contributing to Podverse Next-Gen, please see our [Local Dev Setup documentation](https://github.com/podverse/podverse-ops/blob/v5-develop/dev/local-dev-setup.md). We can also be reached through [Matrix or Discord](https://podverse.fm/contact).

I'm sorry for the abrupt announcement, but thank you for your bug reports and feature requests.
